ipaserver/install/ipa_cacert_manage.py

```diff
@@ -216,7 +216,7 @@ def renew_external_step_2(self, ca, old_cert):
             options.external_cert_files, DN(old_cert.subject))
 
         with open(cert_file.name) as f:
```

Comment:
Hi @stlaz 
thank you for your review. I updated the PR with your proposal and also handled the case where ca_flags.usages is None.
It can be tested with
1/ install with self signed
2/ ipa-cacert-manage renew --external-ca + ipa-certupdate
3/ ipa-cacert-manage renew --self-signed